src/app/personside/infotabs/meldinger/traadliste/TraadListeElement.tsx

```diff
@@ -63,7 +64,7 @@ function TraadListeElement(props: Props) {
     const dyplenker = useInfotabsDyplenker();
 
     useOnMount(() => {
-        if (props.erValgt) {
+        if (props.erValgt && !props.ikkeTaFokus) {
```

Comment:
Hvorfor må denne legges til? Virker litt snålt. :7

Reply:
La inn denne for at ikke fokuset skulle bli flyttet til en melding når du var i søkeruten. Bør nok gjøres på en annen måte